### ISSUE TITLE
0.0.3.201802081723 guava mixup

### ISSUE THREAD

**Issue by steven-aerts** [RepreZen/KaiZen-OpenApi-Parser#130]
In the latest version of the kaizen openapi parser, a new dependency was added to `guava-15.0-cd1.0.jar`.

However guice 4.1.0 is [already depending on](https://mvnrepository.com/artifact/com.google.inject/guice/4.1.0) guava 19.0.

Projects which depend on kaizen will try to download `guava-19.0-cdi1.0.jar`.  So they take the version of guava provided by guice with the classifier specified by kaizen.

This binary of course does not exist.

I propose to remove again the dependency on guava from the pom or line up its version with the one provided by google guice.

**Comment by andylowry** [comment]
@steven-aerts, thanks.

There was an actual reason for this. I'm using JsonOverlay in another context in an Eclipse RCP application. Eclipse is OSGi-based, and it's configured with a singleton guava bundle at v15 (in the Mars.2 distribution that we're currently building for). When I bundle-ize this project, it includes a Guava 19 bundle, which then conflicts with the eclipse environment and can cause problems. Constraining to v15 did not appear to affect my use of guice.

As it turns out, the bundling process includes a guava 19 bundle despite my dependency change, so my attempt to constrain this project isn't helpful there. Instead I need to be careful to exclude that bundle when updating the target platform for my RCP app.

So bottom line is I'll remove the dependency. :)

**Comment by andylowry** [comment]
@steven-aerts I've removed the dependency in master branch (2a6fbbf20905cab606c97da718e05663ec25a0c0). If the guava dependency is causing you any problem using this package, I can push a new version, but otherwise I'll wait and include this with the next set of changes.
